resources/services/compute/compute_instances.go

```diff
@@ -800,8 +800,6 @@ func fetchComputeInstanceSchedulingNodeAffinities(ctx context.Context, meta sche
 }
 func fetchComputeInstanceServiceAccounts(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
 	r := parent.Item.(*compute.Instance)
-	if r.Scheduling != nil {
-		res <- r.ServiceAccounts
-	}
+	res <- r.ServiceAccounts
```

Comment:
What is this change?

Reply:
There's no need to check `r. Scheduling` in this fetch function -- otherwise we don't fetch the `ServiceAccounts` at all

Reply:
@roneli - Was this just a bug that has been present since the beginning or is there an internal dependency where instances must have a schedule in order to have `ServiceAccounts`? Should we also remove the check on line 796?

Reply:
the 796 is actually correct: if the `r.Scheduling` is nil, we'll get panic on `r.Scheduling.NodeAffinities` access